cmake/projects/NLopt/hunter.cmake

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    NLopt
+    VERSION
+    2.5.0
+    URL
+    "https://github.com/t0p4/nlopt/archive/v2.5.0-p0.tar.gz"
```

Comment:
I've created fork https://github.com/hunter-packages/nlopt and branch `hunter-2.5.0`. Please create pull request with patches there, see https://docs.hunter.sh/en/latest/creating-new/patch.html

Reply:
I've made patch form master branch no from tag 2.5.0
What do I need to do&

Reply:
> I've made patch form master branch no from tag 2.5.0

Can it be based on stable release or you want to have some fixes/features from latest master?